app/views/content/life-as-a-teacher.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Life as a teacher"
+description: Deciding if teaching is right for you? Learn how rewarding life as a teacher can be, including pay, benefits, and career progression.
+content:
+image: "static/images/content/hero-images/0029.jpg"
+colour: "yellow"
+layout: "layouts/category"
+title_paragraph: |-
+    Choosing or changing a career can be a big decision. Find out what it's like to be a teacher in England, including how varied and rewarding teaching can be.
```

Comment:
Do we need to mention life as a teacher here, just thinking for consistency, could we have 'Find out what life is like as a teacher in England' etc, just a thought!